indico/modules/rb_new/templates/booking_list.html

```diff
@@ -17,7 +17,7 @@
                 </div>
                 <div class="toolbar">
                     <a href="{{ url_for('rooms_new.roombooking', path='book', **event_rb_params) }}"
-                       target="_blank" class="i-button highlight">
+                       target="_blank" class="i-button highlight {{ 'disabled' if is_past_event }}">
```

Comment:
Ideally that's done for contributions/blocks as well, but maybe checking for the whole event is enough... In that case I'd also disable the session/contribution buttons though...

Reply:
Just the buttons or hide the fields altogether?

Reply:
Disabling with a tooltip is better I guess. But now that you mention hiding.. we could just filter out contributions/blocks that started in the past!